src/experiments/tests/move-permutive-segmentation.ts

```diff
@@ -0,0 +1,29 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const movePermutiveSegmentation: ABTest = {
+	id: 'MovePermutiveSegmentation',
+	author: '@commercial-dev',
+	start: '2025-01-14',
+	expiry: '2025-02-28',
+	audience: 0 / 100,
+	audienceOffset: 20 / 100,
```

Comment:
I know its a 0% test, Wondered what this does?

Reply:
The offset is to make sure that if we set it live (eg not 0%) that the test audience wouldn't overlap with that of your test